src/graphql/schemas/typeDefs/hyperboardTypeDefs.ts

```diff
@@ -0,0 +1,135 @@
+import { Field, ObjectType } from "type-graphql";
+import { BasicTypeDef } from "./baseTypes/basicTypeDef.js";
+import { EthBigInt } from "../../scalars/ethBigInt.js";
+import { User } from "./userTypeDefs.js";
+import { GraphQLBigInt } from "graphql-scalars";
+
+@ObjectType({
+  description: "Hyperboard of hypercerts for reference and display purposes",
+})
+class Hyperboard extends BasicTypeDef {
+  @Field({ description: "Name of the collection" })
+  name?: string;
+  @Field(() => [EthBigInt], {
```

Comment:
here chainID is `bigint | number`, on line 64 they're `bigint` and in `contractTypeDefs` they're `bigint | number | string`. As they're `uint256` they should at least be `bigint` and `string` is the useful intermediate type.

Let's make sure to harmonise the types as this will get weird when building on different types

Reply:
Resolved in https://github.com/hypercerts-org/hypercerts-api/commit/9436dd6af93162b30d97951db3cc68888e364666

Reply:
Missed at least the `User`: https://github.com/hypercerts-org/hypercerts-api/blob/9436dd6af93162b30d97951db3cc68888e364666/src/graphql/schemas/typeDefs/userTypeDefs.ts#L15

Reply:
Updated that one as well.